package.json

```diff
@@ -4,14 +4,16 @@
   "license": "MIT",
   "author": "Akinjide Bankole <r@akinjide.me>",
   "scripts": {
-    "lint": "xo && stylelint extension/*.css",
+    "lint": "xo && stylelint extension/*.css && web-ext lint",
     "test": "npm run lint",
     "test:watch": "ava --verbose --watch",
-    "build": "webpack --mode=production",
+    "build": "webpack --mode=production && web-ext build --source-dir=./extension/build --artifacts-dir=./dist --overwrite-dest",
     "build:watch": "webpack --mode=development --watch --devtool=source-map",
     "semver": "VERSION=$npm_package_version; echo $VERSION; dot-json extension/manifest.json version $VERSION --indent 2",
-    "pre:deploy": "cd extension && webstore upload --source=build --auto-publish",
-    "deploy": "npm-run-all semver build pre:deploy --sequential --print-name"
+    "start:firefox": "web-ext run --source-dir ./extension/build/",
+    "deploy:firefox": "from-env web-ext sign --api-key %AMO_JWT_ISSUER --api-secret %AMO_JWT_SECRET --source-dir=./extension/build --artifacts-dir=./dist --channel=listed",
+    "deploy:chrome": "cd extension && webstore upload --source=build --auto-publish",
+    "deploy": "npm-run-all semver build deploy:chrome deploy:firefox --sequential --print-name"
```

Comment:
run chrome and firefox deploy in parallel `npm-run-all semver build --print-name --parallel deploy:*`